draft-ietf-emu-eap-tls13.xml

```diff
@@ -63,6 +63,8 @@ href='http://xml.resource.org/authoring/rfc2629.xslt' ?>
 
 	<t>This document defines how to use EAP-TLS with TLS 1.3 (or higher) and does not change how EAP-TLS is used with older versions of TLS. It does however provide additional guidance on authorization and resumption for EAP-TLS in general (regardless of the underlying TLS version used). While this document updates EAP-TLS <xref target="RFC5216"/>, it remains backwards compatible with it and existing implementations of EAP-TLS. This document only describes differences compared to <xref target="RFC5216"/>. All message flow are specific to TLS 1.3 and do not apply to TLS 1.2.</t>
 
+	<t>This document specifies EAP-TLS 1.3 and does not specify how other TLS-based EAP methods use TLS 1.3. Based on this document, <xref target="I-D.ietf-emu-tls-eap-types"/> specifies how other TLS-based EAP methods use TLS 1.3.</t>
```

Comment:
Maybe we should leave room for other documents in case TEAP gets move to another document.
"The specification for how other TLS-based EAP methods use TLS 1.3 is left to other documents such as <xref target="I-D.ietf-emu-tls-eap-types"/> ."

Reply:
Agree